locales/ar/filters.json

```diff
@@ -16,7 +16,7 @@
 		"hostlist.4.description": "قائمة لحظر الإعلانات ومنع تتبع المستخدم"
 	},
 	{
-		"hostlist.5.name": "OISD Blocklist Basic",
+		"hostlist.5.name": "OISD Blocklist Small",
```

Comment:
As I recall, it was changed manually. Please revert old name for locales except `en`.